com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameProjectBuilder.java

```diff
@@ -115,6 +96,21 @@ private RandomAccessFile createRandomAccessFile(File handle) throws IOException
 
     @Override
     public Task<Void> create(IResource input) throws IOException, CompileExceptionError {
+
+        // We currently don't have a file mapping with an input -> output for certain files
+        // These should to be setup in the corresponding builder!
+        ProtoBuilder.addMessageClass(".animationsetc", MeshSet.class);
+        ProtoBuilder.addMessageClass(".cubemapc", Cubemap.class);
+        ProtoBuilder.addMessageClass(".fontc", FontMap.class);
+        ProtoBuilder.addMessageClass(".fpc", ShaderDesc.class);
+        ProtoBuilder.addMessageClass(".vpc", ShaderDesc.class);
+        ProtoBuilder.addMessageClass(".goc", PrototypeDesc.class);
+        ProtoBuilder.addMessageClass(".meshc", MeshDesc.class);
+        ProtoBuilder.addMessageClass(".meshsetc", MeshSet.class);
+        ProtoBuilder.addMessageClass(".modelc", Model.class);
+        ProtoBuilder.addMessageClass(".skeletonc", Skeleton.class);
+        ProtoBuilder.addMessageClass(".texturesetc", TextureSet.class);
```

Comment:
These mappings "fell away" during the refactoring.

We want to add another extension->class mapping machanic, that allows us to keep it with the actual implementation. E.g. a "@ProtoBuilder(ext, klass)" attribute on the builder class.
I'll leave that for another day though.